### DISCUSSION TITLE
Restore on startup options

### DISCUSSION THREAD

**Discussion by wigging** [zed-industries/zed#11528] [Feature Requests]
What are the other options for setting `"restore_on_startup": "last_workspace"`? I would like to disable this feature but I don't know what value to provide. I tried `"restore_on_startup": false` but it doesn't work. The documentation on the website doesn't have anything for this setting either.

**Comment by Moshyfawn** [comment]
It's `none`.

You should be able to get autocomplete settings like this.

<img width="699" alt="image" src="https://github.com/zed-industries/zed/assets/16290753/d3a79bc7-9fbf-457c-abb8-fb6c6210c4e0">

**Reply by jordanss1** [reply]
How do you get autocomplete? I don't have anything like that.
